platform/qt/qt.cmake

```diff
@@ -57,7 +57,7 @@ include_directories(
 
 # C++ app
 add_executable(mbgl-qt
-    platform/qt/app/app.qrc
+    platform/qt/common.qrc
```

Comment:
Thanks, removed `platform/qt/qmlapp/source.geojson` and moved `common.qrc` to `platform/qt/resources` folder, removing the need for an `alias` on QRC.